test/testinput/bufr_sfcshp.yaml

```diff
@@ -0,0 +1,75 @@
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "./testdata/ships.20180415_subsampled"
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+
+          stationIdentification:
+            query: "*/RPID"
+
+          longitude:
+            query: "*/CLON"
+
+          latitude:
+            query: "*/CLAT"
+
+          sst:
+            query: "*/SST1"
+            transforms:
+              - offset: -273.15
+
+          waterTemperatureMethod:
+            query: "*/MSST"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testoutput/bufr_sfcship.nc"
+
+      globals:
+        - name: "dataType@MetaData"
+          type: string
+          value: "Surface Ship"
+
+      variables:
+        - name: "datetime@MetaData"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "datetime"
+
+        - name: "stationIdentification@MetaData"
+          source: variables/stationIdentification
+          longName: "Report Identifier"
+          units: "string"
+
+        - name: "latitude@MetaData"
+          source: variables/latitude
+          longName: "Latitude"
+          units: "degrees_north"
+          range: [-90, 90]
+
+        - name: "longitude@MetaData"
+          source: variables/longitude
+          longName: "Longitude"
+          units: "degrees_east"
+          range: [-180, 180]
+
+        - name: "waterTemperatureMethod@MetaData"
+          source: variables/waterTemperatureMethod
+          longName: "Method of Water Temperature Measurement"
+ 
+        - name: "seaSurfaceTemperature@ObsValue"
+          coordinates: "longitude latitude"
+          source: variables/sst
+          longName: "sea_surface_temperature"
+          units: "deg C"
+          range: [-23.0, 40.0]
```

Comment:
Update these:
- name: sea_surface_temperature
  longName: "Sea Surface Temperature"

Reply:
Done. Is there a document of standards for these?

Reply:
@AndrewEichmann-NOAA the best place is the obs files.
Thanks for doing this.